tests/views/adScreen-test.js

```diff
@@ -270,13 +271,44 @@ describe('AdScreen', function() {
       mutingForAutoplay: true
     };
 
-    var wrapper = Enzyme.mount(
+    const wrapper = Enzyme.mount(
       <AdScreen
         controller={mockController}
         skinConfig={defaultSkinConfig}
       />);
-    var unmuteIcons = wrapper.find(UnmuteIcon);
+    const unmuteIcons = wrapper.find(UnmuteIcon);
     expect(unmuteIcons.length).toBe(0);
   });
 
+  it('should hide controlBar after resize', () => {
+    let controlBarVisible = true;
+
+    mockController.startHideControlBarTimer = () => {
```

Comment:
would be great to use `jest.fn()` instead and change the expectation to `expect(mockController.startHideControlBarTimer).toBeCalled()`;

Reply:
\+